src/media/record/record_device.cpp

```diff
@@ -262,6 +262,7 @@ device_serializer::snapshot_collection librealsense::record_device::get_extensio
             case RS2_EXTENSION_COMPOSITE_FRAME : break;
             case RS2_EXTENSION_POINTS          : break;
             case RS2_EXTENSION_RECORD          : break;
+            case RS2_EXTENSION_SAFETY_SENSOR   : break; // TODO - add snapshot for safety sensor
```

Comment:
I see ` void safety_sensor::create_snapshot(std::shared_ptr<safety_sensor>& snapshot) const` next in the code, isn't this the snapshot?

Reply:
ok